src/services/riskAnalysis.services.ts

```diff
@@ -35,7 +50,7 @@ function analyzeSystem(packages: Package[]): number {
  * @param pkg to analyze
  * @param cves list of vulnerabilities in pkg
  */
-function analyzePackage(pkg: Package, cves: Vulnerability[]) {
```

Comment:
clarify in a one liner here the difference between analyzePackage and analyze Vulnerability is - seems like analyze package just calls analyze Vulnerability - in which case its unclear to me why analyzeVuln is called first and then analyzePackage is called (is the same work being repeated?)

Reply:
good catch - this is an error
I'll push a fix